dts/common/nordic/nrf54h20_enga.dtsi

```diff
@@ -0,0 +1,310 @@
+		};
+	};
+
+	clocks {
+		fll16m: fll16m {
+			compatible = "fixed-clock";
+			#clock-cells = <0>;
+			clock-frequency = <DT_FREQ_M(16)>;
+		};
+	};
+
+	soc {
+		#address-cells = <1>;
+		#size-cells = <1>;
+
+		mram1x: mram@e000000 {
+			compatible = "nordic,mram";
+			reg = <0xe000000 DT_SIZE_K(2048)>;
+			write-block-size = <16>;
+		};
+
+		cpuapp_uicr: uicr@fff8000 {
+			compatible = "nordic,nrf-uicr-v2";
+			reg = <0xfff8000 DT_SIZE_K(2)>;
+			domain = <2>;
+			ptr-ext-uicr = <&cpuapp_uicr_ext>;
+		};
+
+		cpurad_uicr: uicr@fffa000 {
+			compatible = "nordic,nrf-uicr-v2";
+			reg = <0xfffa000 DT_SIZE_K(2)>;
+			domain = <3>;
+			ptr-ext-uicr = <&cpurad_uicr_ext>;
+		};
+
+		ficr: ficr@fffe000 {
+			compatible = "nordic,nrf-ficr";
+			reg = <0xfffe000 DT_SIZE_K(2)>;
+			#nordic,ficr-cells = <1>;
+		};
+
+		cpuapp_ram0: sram@22000000 {
+			compatible = "mmio-sram";
+			reg = <0x22000000 DT_SIZE_K(32)>;
+			#address-cells = <1>;
+			#size-cells = <1>;
+			ranges = <0x0 0x22000000 0x8000>;
+		};
+
+		cpurad_ram0: sram@23000000 {
+			compatible = "mmio-sram";
+			reg = <0x23000000 DT_SIZE_K(64)>;
+			#address-cells = <1>;
+			#size-cells = <1>;
+			ranges = <0x0 0x23000000 0x10000>;
+		};
+
+		cpuapp_peripherals: peripheral@52000000 {
+			#address-cells = <1>;
+			#size-cells = <1>;
+			ranges = <0x0 0x52000000 0x1000000>;
+
+			cpuapp_hsfll: clock@d000 {
+				compatible = "nordic,nrf-hsfll";
+				#clock-cells = <0>;
+				reg = <0xd000 0x1000>;
+				clocks = <&fll16m>;
+				clock-frequency = <DT_FREQ_M(320)>;
+				nordic,ficrs =
+					<&ficr NRF_FICR_TRIM_APPLICATION_HSFLL_TRIM_VSUP>,
+					<&ficr NRF_FICR_TRIM_APPLICATION_HSFLL_TRIM_COARSE_0>,
+					<&ficr NRF_FICR_TRIM_APPLICATION_HSFLL_TRIM_FINE_0>;
+				nordic,ficr-names = "vsup", "coarse", "fine";
+			};
+		};
+
+		cpurad_peripherals: peripheral@53000000 {
+			#address-cells = <1>;
+			#size-cells = <1>;
+			ranges = <0x0 0x53000000 0x1000000>;
+
+			cpurad_hsfll: clock@d000 {
+				compatible = "nordic,nrf-hsfll";
+				#clock-cells = <0>;
+				reg = <0xd000 0x1000>;
+				clocks = <&fll16m>;
+				clock-frequency = <DT_FREQ_M(256)>;
+				nordic,ficrs =
+					<&ficr NRF_FICR_TRIM_RADIOCORE_HSFLL_TRIM_VSUP>,
+					<&ficr NRF_FICR_TRIM_RADIOCORE_HSFLL_TRIM_COARSE_1>,
+					<&ficr NRF_FICR_TRIM_RADIOCORE_HSFLL_TRIM_FINE_1>;
+				nordic,ficr-names = "vsup", "coarse", "fine";
+			};
+		};
+
+		global_peripherals: peripheral@5f000000 {
+			#address-cells = <1>;
+			#size-cells = <1>;
+			ranges = <0x0 0x5f000000 0x1000000>;
+
+			cpuppr_vpr: vpr@908000 {
+				compatible = "nordic,vpr-coprocessor";
+				reg = <0x908000 0x1000>;
+				status = "disabled";
+				cpu = <13>;
+				#address-cells = <1>;
+				#size-cells = <1>;
+				ranges = <0x0 0x908000 0x4000>;
+
+				cpuppr_clic: interrupt-controller@1000 {
+					compatible = "nordic,nrf-clic";
+					reg = <0x1000 0x3000>;
+					status = "disabled";
+					#interrupt-cells = <2>;
+					interrupt-controller;
+					#address-cells = <1>;
+				};
```

Comment:
Not sure if this is the right place for this node.
Maybe `soc` in `dts/riscv/nordic/nrf54h20_enga_cpuppr.dtsi` would be better?

Reply:
@57300 ^

Reply:
I'm not sure what would be the benefit in routing it in owner's DT instead of VPR's DT - but I might be missing something specific to nRF54H architecture.
Not a blocker from my side anyway - we will revisit it when adding VPR to nRF54L.